handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -120,7 +120,7 @@
         } else {
             // Test if netty-tcnative is in the classpath first.
             try {
-                Class.forName("io.netty.internal.tcnative.SSLContext", false, OpenSsl.class.getClassLoader());
+                Class.forName("io.netty.internal.tcnative.SSLContext", false, PlatformDependent.getClassLoader(OpenSsl.class));
```

Comment:
Fix check style please `Line is longer than 120 characters`

Reply:
Thanks, i have done